xbmc/music/Album.h

```diff
@@ -71,6 +71,8 @@ class CAlbum
   }
 
   std::string GetArtistString() const;
+  const std::vector<std::string> GetAlbumArtist() const;
+  const std::vector<std::string> GetMusicBrainzAlbumArtistID() const;
```

Comment:
I would love to see brief explanations here. I think they improve our codebase. Just a suggestion, nothing you really have to do to get this merged.